src/sbvr-api/permissions.ts

```diff
@@ -518,7 +518,11 @@ const generateConstrainedAbstractSql = (
 		odata,
 	);
 
-	_.set(odata, ['tree', 'options', '$filter'], collapsedPermissionFilters);
+	_.set(
+		odata,
+		['tree', 'options', '$filter'],
+		collapsedPermissionFilters || false,
```

Comment:
As a general rule it's much better to use `??` if you're only planning to default null/undefined as it makes the intent much clearer and avoids questions around whether it was intentional to default all falsey values or an oversight, particularly if the types change due to other later changes